docs/language/access-control.md

```diff
@@ -30,7 +30,7 @@ will be covered in a later section.
 Top-level declarations
 (variables, constants, functions, structures, resources, interfaces)
 and fields (in structures, and resources) are always only able to be written
-to in the scope where it is defined (self).
+to and mutated in the scope where it is defined (self).
```

Comment:
Would it be useful to clarify what mutate means in this context? Some less-experienced developers might not fully understand what it means and need some clarification.